cmd/root.go

```diff
@@ -59,6 +59,7 @@ func Execute() {
 	}
 	err := segmentClient.Close()
 	if err != nil {
+		_,_ = fmt.Fprintln(os.Stderr,"Failed to Close Segment Client: ", err)
```

Comment:
What are we doing here?
Should it not be logged ?

Reply:
@rafiu007 I thought of printing the error to Stderr since that is the way errors have been printed in that particular file. Should I change to logging the error?

Reply:
@jparsai what do u think?

Reply:
anything is fine as long as we get actual error in CLI response.